src/pages/Chat/Chat.styled.jsx

```diff
@@ -9,19 +9,21 @@ export const ChatLayoutContainer = styled.div`
 `
 
 export const HeaderChat = styled.div`
-  ${[rounded({ bl: '25px', br: '25px' }), p({ all: '1.5%' }), size({ height: '11vh' }), bg.primary]}
+  ${[rounded({ bl: '25px', br: '25px' }), p({ all: '1.5%' }), size({ height: '11vh' })]}
   display: flex;
   flex-direction: row;
   align-items: center;
   position: fixed;
+  background-color: var(--light-blue-12);
   width: 100%;
   grid-row: 1/2;
   height: 80px;
 `
 
 export const HeaderChatImgContainer = styled.div`
-  width: 33.33vw;
+  //width: 33.33vw;
   text-align: right;
+  margin-left: -80px;
```

Comment:
What does this line? I check and removing it the header is centered

![imagen](https://user-images.githubusercontent.com/66505715/156257279-dbc80b0e-14a9-4edd-9526-17e5347c92cc.png)

But adding it the header is not centered

![imagen](https://user-images.githubusercontent.com/66505715/156257324-47fd965b-b92f-4250-9143-0c81bfe1fd50.png)

I think it's bad to abuse of margin

Reply:
To make the image and text next to the back button you should reduce the width of the backbutton. 

How you notice, the width is 33vw
![imagen](https://user-images.githubusercontent.com/66505715/156501669-2da5e7f1-2853-4543-8c30-38d9baf51fad.png)

You can to use one media query to change this width in small screens
![imagen](https://user-images.githubusercontent.com/66505715/156501781-95731284-383e-4647-b759-4210652b30f7.png)

If you remove the width to 0, then the text and image will be next to the back button